test/server/Server.test.js

```diff
@@ -301,12 +301,12 @@ describe('Server', () => {
       });
     });
 
-    it('should always allow any host if options.firewall is disabled', () => {
+    it('should always allow any host if options.allowedHosts is enabled', () => {
```

Comment:
Let's move test for `allowedHosts` to own test file, in `Server.test.js` should be only API tests in future

Reply:
yep lets do it

Reply:
done @alexander-akait